alcs-frontend/src/app/services/application/application.service.ts

```diff
@@ -70,9 +70,7 @@ export class ApplicationService {
   }
 
   private async fetchCodes() {
-    const codes = await firstValueFrom(
-      this.http.get<ApplicationMasterCodesDto>(`${environment.apiUrl}/application-code`)
-    );
+    const codes = await firstValueFrom(this.http.get<ApplicationMasterCodesDto>(`${environment.apiUrl}/code`));
     this.statuses = codes.status;
     this.$applicationStatuses.next(this.statuses);
```

Comment:
Should this be $cardStatuses now or are ApplicationStatus separate from ReconStatus?